controller/pom.xml

```diff
@@ -147,6 +147,11 @@
            <artifactId>junit</artifactId>
            <scope>test</scope>
         </dependency>
+        <dependency>
+           <groupId>org.mockito</groupId>
+           <artifactId>mockito-core</artifactId>
+           <scope>test</scope>
+        </dependency>
```

Comment:
We are planning to move all test dependencies to the testbom. I think it reasonable to merge this as it is. 

@parsharma We will need to take into account this new test dependency and move it to the testbom. Sadly your PR is still in the queue since we need to get the WildFly Full counterpart merged before merging yours.